src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -222,6 +273,32 @@ public class BssEpisode
     public string? final_action_code { get; set; }
 }
 
+public class BssSubject
```

Comment:
Can we move these BssEpisode, BssSubject and FileType each to a separate file

Reply:
Ready for review